rfcs/0166-nix-formatting.md

```diff
@@ -398,31 +421,28 @@ These limits should be configurable, and the soft length limit should default to
         a = foo
           bar
           baz;
-      }; # <-- The decrease by two levels here is okay though
+      }; # <-- The decrease by two levels here is okay
     in
     null
     ```
 
 ### Expansion of expressions
 
-**Description**
-
-Unless stated otherwise, any expression that fits onto one single line will be trivially formatted as such.
+Unless stated otherwise, any expression that fits onto one single line must be trivially formatted as such.
 
-For sequences of items in expressions, like elements in a list, key-value pairs in attribute sets, or function arguments, the following applies:
+For list elements, attributes and function arguments, the following applies:
```

Comment:
An Oxford comma here would prevent someone from thinking ‘list elements’ is a concept that expands to ‘attributes and function arguments’.
```suggestion
For list elements, attributes, and function arguments, the following applies:
```

Reply:
Sounds good!